UefiTestingPkg/AuditTests/PagingAudit/Windows/MemoryRangeObjects.py

```diff
@@ -91,9 +97,10 @@ def __init__(self, record_type, *args, **kwargs):
         self.Attribute = 0
         self.AddressBitwidth = None
         self.PageSplit = False
+        self.CpuNumber = None
 
         # Check to see whether we're a type that we recognize.
-        if self.RecordType not in ("TSEG", "MemoryMap", "LoadedImage", "SmmLoadedImage", "PDE", "GDT", "IDT", "PTEntry", "TTEntry", "MAT", "GuardPage", "Bitwidth"):
+        if self.RecordType not in ("TSEG", "MemoryMap", "LoadedImage", "SmmLoadedImage", "PDE", "GDT", "IDT", "PTEntry", "TTEntry", "MAT", "GuardPage", "Bitwidth", "Stack", "StackGuard", "ApStack", "ApStackGuard", "ApSwitchStack", "Null"):

```

Comment:
This condition would be more readable and maintainable if it was defined outside the condition itself (ie. assigned a name and maintained with other data structures).

Not necesarily related to your changes here but the changes do increase the length.

Reply:
Moved to the else case. Please take another look

Reply:
Looks good